wideq/client.py

```diff
@@ -349,8 +353,26 @@ def enum_name(self, key, value):
         """Look up the friendly enum name for an encoded value.
         """
         options = self.value(key).options
+        if value not in options:
+            logging.warning(
+                'Value `%s` for key `%s` not in options: %s. Values from API: '
+                '%s', value, key, options, self.data['Value'][key]['option'])
+            return _UNKNOWN
         return options[value]
 
```

Comment:
Looks up a reference type, this just refers to a key at the same level as the `Value` key in the json response.